src/builders/skopeo_builder.rs

```diff
@@ -526,9 +526,129 @@ impl SkopeoSyslinuxBuilder {
+                    "scripts/config",
+                    "--enable",
+                    "CONFIG_SQUASHFS_CHOICE_DECOMP_BY_MOUNT",
+                ],
+                false,
+            )
+            .context(
+                "Failed to enable CONFIG_SQUASHFS_CHOICE_DECOMP_BY_MOUNT flag to kernel config",
+            )?;
+            Self::run_command(
+                &[
+                    "scripts/config",
+                    "--enable",
+                    "CONFIG_SQUASHFS_MOUNT_DECOMP_THREADS",
+                ],
+                false,
+            )
+            .context(
+                "Failed to enable CONFIG_SQUASHFS_MOUNT_DECOMP_THREADS flag to kernel config",
+            )?;
+            Self::run_command(
+                &["scripts/config", "--enable", "CONFIG_SQUASHFS_XATTR"],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_XATTR flag to kernel config")?;
+            Self::run_command(
+                &["scripts/config", "--enable", "CONFIG_SQUASHFS_ZLIB"],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_ZLIB flag to kernel config")?;
+            Self::run_command(
+                &["scripts/config", "--enable", "CONFIG_SQUASHFS_LZ4"],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_LZ4 flag to kernel config")?;
+            Self::run_command(
+                &["scripts/config", "--enable", "CONFIG_SQUASHFS_LZO"],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_LZO flag to kernel config")?;
+            Self::run_command(&["scripts/config", "--enable", "CONFIG_SQUASHFS_XZ"], false)
+                .context("Failed to enable CONFIG_SQUASHFS_XZ flag to kernel config")?;
+            Self::run_command(
+                &["scripts/config", "--enable", "CONFIG_SQUASHFS_ZSTD"],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_ZSTD flag to kernel config")?;
+            Self::run_command(
+                &[
+                    "scripts/config",
+                    "--enable",
+                    "CONFIG_SQUASHFS_4K_DEVBLK_SIZE",
+                ],
+                false,
+            )
+            .context("Failed to enable CONFIG_SQUASHFS_4K_DEVBLK_SIZE flag to kernel config")?;
+            Self::run_command(
+                &[
+                    "scripts/config",
+                    "--set-val",
+                    "3",
+                    "CONFIG_SQUASHFS_FRAGMENT_CACHE_SIZE",
+                ],
+                false,
+            )
+            .context("Failed to set CONFIG_SQUASHFS_FRAGMENT_CACHE_SIZE value to kernel config")?;
+            Self::run_command(
+                &["scripts/config", "--disable", "CONFIG_SQUASHFS_EMBEDDED"],
+                false,
+            )
+            .context("Failed to disable CONFIG_SQUASHFS_EMBEDDED flag to kernel config ")?;
```

Comment:
This is ugly as hell but does the job. We should probably come up with some neat system where we could easily describe feature sets and give them names. Like name of the feature set `enable_squashfs` and then some table where above enable/disable/set combinations are easily described.